src/main/java/catalinazoldiSeleniumBasics2/driver/BrowserManager.java

```diff
@@ -0,0 +1,48 @@
+package catalinazoldiSeleniumBasics2.driver;
```

Comment:
Please create a `teorie `package in `src/main/java/catalinazoldi` package and move this BrowserManager class there. After you move the class, please remove `catalinazoldiSeleniumBasics2` package from `src/main/java`. Thanks!